pyproject.toml

```diff
@@ -0,0 +1,121 @@
+[build-system]
+requires = [
+  "setuptools", 
+  "wheel",
+  "imagecodecs>=2021.2.26",
```

Comment:
Upto you, but I would probably add a comment here saying that the runtime dependencies have been added as build-time dependencies, simply due to the fact that [src/bfio/__init__.py exports some aliases](https://github.com/PolusAI/bfio/pull/68/files#diff-56f4b8a4749625a073696f969c033711fd761ffcb170261dc3bbc368719c72d6R29), which causes cascading imports which effectively requires adding all runtime dependencies at build-time.

Reply:
will do that!